### PR TITLE
ASC-1029 Automate converge steps on OSP13

### PR DESCRIPTION
**Pull request by johnduarte**

### CODE REVIEW

**Comment by johnduarte** [comment]
This change has been tested in conjunction with the change in https://github.com/rcbops/rpc-openstack-system-tests/pull/476 and successfully concluded the converge step against an OSP13 MNAIO deployment.

System-tests SHA
```
root@jrd-osp13-b:~/sys-tests# git rev-parse HEAD
6bcdae871b05a5d21b8ad631f93eb53044472f62
```
Post-deploy molecule SHA
```
root@jrd-osp13-b:~/sys-tests/molecules/molecule-rpc-openstack-post-deploy# git rev-parse HEAD
e500840783cbfa13d3c34700ba186ffccae0e277
```
Results
```
    PLAY RECAP *********************************************************************
    director                   : ok=44   changed=16   unreachable=0    failed=0
```

File: tasks/create_clouds.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      {{
+        overcloudrc_data | default({}) |
+        combine({ item.split('=')[0] |
+                  regex_replace('export\s+'): item.split('=')[1] })
+      }}
+  with_items: "{{ overcloudrc.stdout_lines }}"
+
+- name: Debug overcloudrc_data
+  debug:
+    var: overcloudrc_data
```
Comment (ryan-rs) [inline tasks/create_clouds.yml]: You left a debug message here, is this on purpose?

File: tasks/set_facts.yml [hunk 2]
```diff
@@ -57,12 +57,12 @@
 - name: Set cli_host
   set_fact:
     cli_host: "{{ ansible_hostname }}"
-    #cli_host: "{{ utility_container if utility_container is defined else ansible_hostname }}"
+    # cli_host: "{{ utility_container if utility_container is defined else ansible_hostname }}"
```
Comment (ryan-rs) [inline tasks/set_facts.yml]: Is this comment for clarity? Otherwise it is dead code.

File: tasks/create_clouds.yml (hunk 1 above)
Comment (johnduarte) [reply tasks/create_clouds.yml]: Yes, this task is here on purpose. This data is generated from a regex pattern that will break if the way that os writes the openrc file changes. This statement will be helpful in determining when the gathered data is not as expected.

File: tasks/set_facts.yml (hunk 2 above)
Comment (johnduarte) [reply tasks/set_facts.yml]: Update pushed to remove this comment.

**Pull request merged**